### PR TITLE
components - inputStyleNeutral: Remove deprecated `reducedMotion` util.

### PR DESCRIPTION
**Pull request by DaniGuardiola**
Remove `reducedMotion` since it is deprecated now. Parent issue: #60902

### CODE REVIEW

File: packages/components/src/utils/input/base.js [hunk 1]
```diff
@@ -15,7 +14,10 @@ export const inputStyleNeutral = css`
 	transition: box-shadow 0.1s linear;
```
Comment (tyxla) [inline packages/components/src/utils/input/base.js]: Did we mean to remove this line?

Comment (DaniGuardiola) [reply packages/components/src/utils/input/base.js]: Another good catch :)

**Review by tyxla — APPROVED** [review]
LGTM, but you'll need to update some snapshots 👍

**Comment by tyxla** [comment]
Also, @DaniGuardiola, note that when committing in Gutenberg you should add the WP.org contributors to your commit message, see https://github.com/WordPress/gutenberg/pull/61122#issuecomment-2078066517

**Pull request merged**